psse/psse-converter/src/main/java/com/powsybl/psse/converter/LineConverter.java

```diff
@@ -190,10 +200,25 @@ static void update(Network network, PssePowerFlowModel psseModel) {
         });
     }
 
+    private String getName() {
+        String name = "";
+        if (version.major() == V32 || version.major() == V33) {
+            PsseBus busI = busNumToPsseBus.get(psseLine.getI());
+            PsseBus busJ = busNumToPsseBus.get(psseLine.getJ());
+            if (busI != null && busJ != null) {
+                name = String.format("%s_%s_%s", busI.getName().trim(), busJ.getName().trim(), psseLine.getCkt());
+            }
+        } else if (version.major() == V35) {
+            name = psseLine.getName();
```

Comment:
When the name contains only white chars, it would be better to return `null` (see comments on this PR).